docs/src/components/Navbar/index.tsx

```diff
@@ -44,7 +44,6 @@ const StyledWrapper = styled.header`
 const StyledInner = styled.div`
   max-width: ${MAX_CONTENT_WIDTH};
   padding: 0 ${CONTENT_PADDING};
-  box-sizing: content-box;
```

Comment:
If we remove this, I also don't think it gets according to the designs 🤔 it gets smaller than the content in the homepage

Reply:
now it's correct, in Figma it's 1440px including paddings, before it was bigger.

Reply:
changed the size of HeaderBox, so it's visually ok